task_keer/docker_overview_keer/docker_keer

```diff
@@ -0,0 +1,118 @@
+#Docker 
+##Docker Overview
+- Open PLatform
+- Life cyle of Docker
+1. Build: Docker build command is used after the application is configured 
+2. Shipping:Push of image making it available centrally 
+- using a binary repository[Docker Trusted Registory(DTR)]
+3. running application 
+- Docker has Docker Engine {light weighted container runtime +packaging tool for docker image } 
+..................................................................................................
+##Objective 
+- To reduce the error prone 
+- To ease the maintance 
+- To elimante individual server need 
+..................................................................................................
+- - Properties of docker:Isolation and portability
+- - Dockerised apps are portable
+- - Reduces gap between development, QA & production
+- - Usage: system admistrators-to provide standard environment
+- - Tracks of version of Images and Containers using git 
+- - Docker hub -private and public 
+..................................................................................................
+##Benifts of Docker 
+- 1.Simplicity 
+- 2. Speed
+..................................................................................................
+##Run command 
+- docker run -d -p 80:80 docker/getting-started
+- d to run the container in the detached mode a
```

Comment:
what is -p?
mode a? means in the above usage?
what is 80:80?
can you please explain